include/triggeralgs/LowEnergyEvent/TriggerActivityMakerLowEnergyEvent.hpp

```diff
@@ -0,0 +1,154 @@
+/**
+ * @file TriggerActivityMakerLowEnergyEvent.hpp
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2021.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef TRIGGERALGS_LOWENERGYEVENT_TRIGGERACTIVITYMAKERLOWENERGYEVENT_HPP_
+#define TRIGGERALGS_LOWENERGYEVENT_TRIGGERACTIVITYMAKERLOWENERGYEVENT_HPP_
+
+#include "detchannelmaps/TPCChannelMap.hpp"
+#include "triggeralgs/TriggerActivityMaker.hpp"
+#include <fstream>
+#include <vector>
+
+#include <chrono>
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_set>
+#include <utility>
+
+namespace triggeralgs {
+class TriggerActivityMakerLowEnergyEvent : public TriggerActivityMaker
+{
+
+public:
+  void operator()(const TriggerPrimitive& input_tp, std::vector<TriggerActivity>& output_ta);
+
+  void configure(const nlohmann::json& config);
+
+private:
+  class Window
```

Comment:
Should this be a class of its own? Looks like duplicated code

Reply:
The nested `Window` classes have been separated from the TAMaker and TCMaker classes, and are now defined in `triggeralgs/TPWindow.hpp` and `triggeralgs/TAWindow.hpp` . The algorithms that I have worked on now include this header/implementation file in their `TAMaker` and `TCMaker` classes instead of using duplicated code.